src/adapters/codemirror-6-adapter.ts

```diff
@@ -130,9 +134,9 @@ export class CodeMirror6Adapter implements AdapterInterface {
     switch (format) {
       case 'XML':
       case 'HTML':
-        return _.escape;
+        return encode;
```

Comment:
I'm wondering if this is converting/escaping other characters that lodash did not and if that can cause any issues. Hopefully not, just taking a note here.